install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -366,6 +366,9 @@ data:
 {{- if .Values.global.nodePort.mode }}
   node-port-mode: {{ .Values.global.nodePort.mode | quote }}
 {{- end }}
+{{- if .Values.global.nodePort.enableHealthCheckNodeport }}
```

Comment:
Nit: WDYT about renaming to `.Values.global.nodePort.enableHealthCheck`?

Reply:
Done